packages/api/src/webhooks/cannon.ts

```diff
@@ -626,6 +623,48 @@ export default class WebhookCannon {
   }
 }
 
+export async function resendWebhook(
+  webhook: DBWebhook,
+  webhookResponse: WebhookResponse
+) {
+  const triggerTime = Date.now();
+  const startTime = process.hrtime();
+  let resp: Response;
+  let responseBody: string;
+  let statusCode: number;
+  let errorMessage: string;
+  try {
+    resp = await fetchWithTimeout(webhookResponse.request.url, {
+      method: webhookResponse.request.method,
+      headers: webhookResponse.request.headers,
```

Comment:
I'm not sure if we should repeat the webhook signature here. We add a timestamp to the signature payload to avoid the possibility of replay attacks, so the webhook handler can check that and make sure it is never reused. But then we are doing a replay "attack" ourselves through the `resend` flow!

OTOH this is a "debug-only" API right? We don't expect users to be calling this on a healthy workflow, and maybe they _do_ want the absolute exact payload.

WDYT? Do we have any reference from other platforms? Might be worth expliciting this behavior on the docs, regardless of which one we choose.

Reply:
As discussed, regenerating the sig makes sense 👍